inst/qml/cohenFleissKappa.qml

```diff
@@ -0,0 +1,101 @@
+//
+// Copyright (C) 2013-2020 University of Amsterdam
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public
+// License along with this program.  If not, see
+// <http://www.gnu.org/licenses/>.
+//
+
+import QtQuick 			2.8
+import JASP.Controls 	1.0
+import JASP.Theme		1.0
+import JASP.Widgets 	1.0
+
+Form
+{
+	VariablesForm
+	{
+		height: 300
+
+		AvailableVariablesList { name: "allVariablesList" }
+
+		AssignedVariablesList
+		{
+			name: 			"variables"
+			title: 			qsTr("Variables")
+			allowedColumns: ["nominal", "nominalText", "ordinal"]
```

Comment:
I wonder if we should not also allow continuous data, somehow jasp sometimes reads even ordinal data as continuous, see for example the ICC example in the data library. of course you could change the scaling of the columns, but I wonder what we would lose if we allowed continuous data?

Reply:
I agree. While continuous data will internally be treated as ordinal, there should not be a problem with allowing it as input.